plugin.json

```diff
@@ -21,7 +21,7 @@
         "settings": [{
             "key": "CustomAttributes",
             "type": "custom", 
-            "display_name": "Custom Attributes"
+            "display_name": "Custom Attributes:"
```

Comment:
I wonder if the product should add this instead of having to expect all plugins to add the colon to be consistent?

Reply:
Would having this in a style guide/in the app development workflow be helpful at all, or should it be more automated? And who would be the best person to do the latter? :)

Reply:
@marianunez, this is a good catch and proposal.  Probably a UX consideration also?  @abhijit-singh?